### PR TITLE
Update the release creation documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/developer/build_release.rst.tmpl [hunk 1]
```diff
@@ -19,175 +19,165 @@ the first bug fix release, and ``<version>`` can be ``2.0``, ``2.1``, ...
...
-  to the upper branches i.e.: ``2.2`` => ``2.3``, ``2.3`` => ``2.4`` (master).
-
-  .. note::
-
-     On ``c2cgeoportal`` merge, see if an alembic merge should be done:
+Tasks to do
+-----------
 
-     .. prompt:: bash
+On franch creation (start of the integration phase):
```
Comment (jwkaltz) [inline doc/developer/build_release.rst.tmpl:25]: franch -> branch?
But then it is not clear why we do a branch creation.

File: doc/developer/build_release.rst.tmpl [hunk 2]
```diff
@@ -19,175 +19,165 @@ the first bug fix release, and ``<version>`` can be ``2.0``, ``2.1``, ...
...
+* Use the new demo
 
-        alembic --config=commons/alembic.ini \
-            --name=[main|static] \
-            merge --message="Merge <src> and <dst> branches" \
-            <rev 1> <rev 2>
+.. note::
 
-     Remove the import and replace the core of the method by ``pass`` in the generated file.
+   I don't specify but all your changes sould be commited
```
Comment (jwkaltz) [inline doc/developer/build_release.rst.tmpl:46]: I am not sure to understand. Do we want to say "Do not forget to commit all your changes." ?

File: doc/developer/build_release.rst.tmpl [hunk 3]
```diff
@@ -19,175 +19,165 @@ the first bug fix release, and ``<version>`` can be ``2.0``, ``2.1``, ...
...
+        "$(gopass gs/ci/transifex/rw/username)" \
+        "$(gopass gs/ci/transifex/rw/password)"
 
-    git add .github/workflows/ci.yaml
-    git commit -m "Start version <version + 1>"
+Configure dependabot
+--------------------
 
-Push your changes:
+On ``c2cgeoporal`` and ``ngeo`` master branches, configure the updates for the security issues, see also:
```
Comment (jwkaltz) [inline doc/developer/build_release.rst.tmpl:102]: c2cgeoporal -> c2cgeoportal

File: doc/developer/build_release.rst.tmpl [hunk 4]
```diff
@@ -19,175 +19,165 @@ the first bug fix release, and ``<version>`` can be ``2.0``, ``2.1``, ...
...
+`c2cgeoportal <https://github.com/camptocamp/c2cgeoportal/commit/7783a2d1d02cc58da7cb1e3b9d32a1358cc30524>`_
```
Comment (jwkaltz) [inline doc/developer/build_release.rst.tmpl:103]: It is not clear to me why we are referencing this commit

File: doc/developer/build_release.rst.tmpl [hunk 5]
```diff
@@ -19,175 +19,165 @@ the first bug fix release, and ``<version>`` can be ``2.0``, ``2.1``, ...
...
+`ngeo <https://github.com/camptocamp/ngeo/commit/1d1fe0ee600229daab4a85b75529cbdb8beecd42>`_
 
-.. prompt:: bash
+Configure the rebuild
+---------------------
 
-    git push origin master
+On ``c2cgeoportal`` copy the files ``.github/workflows/ci.yaml`` and ``.github/workflows/qgis.yaml`` from
+new version branche to master branch as ``.github/workflows/rebuild-<new version>.yaml`` and
```
Comment (jwkaltz) [inline doc/developer/build_release.rst.tmpl:110]: branche -> branch

File: doc/developer/build_release.rst.tmpl [hunk 6]
```diff
@@ -19,175 +19,165 @@ the first bug fix release, and ``<version>`` can be ``2.0``, ``2.1``, ...
...
+``.github/workflows/rebuild-qgis-<new version>.yaml`` and do the follownig changes:
```
Comment (jwkaltz) [inline doc/developer/build_release.rst.tmpl:111]: follownig -> following

File: doc/developer/build_release.rst.tmpl [hunk 7]
```diff
@@ -19,175 +19,165 @@ the first bug fix release, and ``<version>`` can be ``2.0``, ``2.1``, ...
...
+   +          ref: ${{ env.MAIN_BRANCH }}
 
-    git fetch
-    git checkout <version>
-    git reset --hard origin/<version>
+Use the ``ngeo`` package linked to the branch
+---------------------------------------------
 
-Tag the new release:
+In ``c2cgeoporatal`` new version branch, in the file ``geoportal/package.json``, the the ``ngeo`` version to
```
Comment (jwkaltz) [inline doc/developer/build_release.rst.tmpl:140]: c2cgeoporatal -> c2cgeoportal
the the -> set the

File: doc/developer/build_release.rst.tmpl [hunk 8]
```diff
@@ -19,175 +19,165 @@ the first bug fix release, and ``<version>`` can be ``2.0``, ``2.1``, ...
...
+Reset the changelog
+-------------------
 
-    git tag <release>
-    git push origin <release>
+On the ``c2cgeoportal`` new version branch:
 
-Run a new job for the <version> branch on Jenkins.
+* Empty the file ``CHANGELOG``
+* Set hte content of the file ``ci/changelog.yaml`` to:
```
Comment (jwkaltz) [inline doc/developer/build_release.rst.tmpl:149]: hte -> the

File: doc/developer/build_release.rst.tmpl (hunk 2 above)
Comment (sbrunner) [reply doc/developer/build_release.rst.tmpl:46]: "I don't specify but all changes should be committed."

Is it clearer?

File: doc/developer/build_release.rst.tmpl (hunk 4 above)
Comment (sbrunner) [reply doc/developer/build_release.rst.tmpl:103]: To give an example on how to configure Dependabot for the new branch...

File: doc/developer/build_release.rst.tmpl (hunk 2 above)
Comment (jwkaltz) [reply doc/developer/build_release.rst.tmpl]: No I don't think so... what is it that do you not specify?

File: doc/developer/build_release.rst.tmpl (hunk 4 above)
Comment (jwkaltz) [reply doc/developer/build_release.rst.tmpl:103]: How about:
..., add in the dependabot configuration the new GMF version to receive updates for security issues; see for example this commit:

Comment (sbrunner) [reply doc/developer/build_release.rst.tmpl:103]: Or:
On ``c2cgeoportal`` and ``ngeo`` master branches, add in the Dependabot configuration the new branch to
receive updates for security issues; see for example those commits:

File: doc/developer/build_release.rst.tmpl (hunk 2 above)
Comment (sbrunner) [reply doc/developer/build_release.rst.tmpl]: Every time you have to commit changes.

Comment (jwkaltz) [reply doc/developer/build_release.rst.tmpl]: So why don't we say "All changes should be committed" or "Do not forget to commit all your changes"

"I don't specify" sounds really strange in a doc.

**Pull request merged**